<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WVent</class>
 <widget class="QWidget" name="WVent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="in_vent_type">
       <property name="text">
        <string>Cooling Duct Shape:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_vent_type"/>
     </item>
     <item>
      <widget class="QLabel" name="in_Alpha0">
       <property name="text">
        <string>Alpha0 :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FloatEdit" name="lf_Alpha0">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_Alpha0_unit">
       <item>
        <property name="text">
         <string>[rad]</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>[Â°]</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="w_vent" native="true">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpButton</class>
   <extends>QLabel</extends>
   <header>......GUI/Tools/HelpButton.h</header>
  </customwidget>
  <customwidget>
   <class>FloatEdit</class>
   <extends>QLineEdit</extends>
   <header>......GUI/Tools/FloatEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
